docs/HISTORY.txt

```diff
@@ -4,7 +4,7 @@ Changelog
 2.0.3 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Extend readme about how use embed-resource with multiple coulors [Nachtalb]
```

Comment:
I'm wondering if this should be a released as a new version? What do you mean, @jone?